### PR TITLE
[INLONG-4400][SDK] Unified dataproxy sdk clusterId property name

### LINKED ISSUE #4400: [Improve][SDK] Unified dataproxy sdk clusterId property name
### Description

In .local and .managerip files, named clusterId, in .local file, named cluster_id, so i want to unified dataproxy sdk clusterId property name.
image

### InLong Component

InLong SDK

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by xuesongxs**
Fixes #4400 

### Documentation
image

In official document, the "cluster_id" needs to be change to "clusterId".

### CODE REVIEW

File: inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java [hunk 1]
```diff
@@ -588,8 +588,8 @@ private ProxyConfigEntry getLocalProxyListFromFile(String filePath) throws Excep
         Map<String, Integer> streamIdMap = getStreamIdMap(localProxyAddrJson);
         proxyEntry.setGroupIdNumAndStreamIdNumMap(groupIdNum, streamIdMap);
         proxyEntry.setLoad(load);
-        if (localProxyAddrJson.has("cluster_id")) {
-            proxyEntry.setClusterId(localProxyAddrJson.get("cluster_id").getAsString());
+        if (localProxyAddrJson.has("clusterId")) {
```
Comment (baomingyu) [inline inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java:591]: Please modify the demo such as TcpClientExample、HttpClientExample and the instructions in the documentation at the same time.

**Review by baomingyu — CHANGES_REQUESTED** [review]

File: inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java (hunk 1 above)
Comment (baomingyu) [reply inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java:591]: demo package is org.apache.inlong.sdk.dataproxy.example

Comment (baomingyu) [reply inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java:591]: The github project where the documentation is located is: https://github.com/apache/incubator-inlong-website

**Review by dockerzhang — APPROVED** [review]

**Review by baomingyu — APPROVED** [review]
+1

**Pull request merged**